### ISSUE TITLE
Add Fooocus Example

### ISSUE THREAD

**Issue by user4707** [modal-labs/modal-examples#711]
Please add an example to use [Fooocus](https://github.com/lllyasviel/Fooocus) with Modal.

Here a basic script I wrote (inspired from A1111 example) but I get timeout error when visiting deployed URL. Model is predownloaded to avoid downloads during cold starts. If anyone can help improve this script and fix timeout errors...

```python
from modal import Image, Stub, gpu, web_server

DOCKER_IMAGE = "nvidia/cuda:12.3.1-base-ubuntu22.04"
PYTHON_VER = "3.10"
GPU_CONFIG = gpu.T4()
PORT = 8000

def init_Fooocus():
    import os
    import subprocess

    os.chdir(f"/Fooocus")
    os.system(f"pip install -r requirements_versions.txt")
    os.chdir(f"./models/checkpoints")
    subprocess.run(f"wget -O juggernautXL_v8Rundiffusion.safetensors 'https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/juggernautXL_v8Rundiffusion.safetensors'", shell=True)


image = (
    Image.from_registry(DOCKER_IMAGE, add_python=PYTHON_VER)
    .run_commands("apt update -y")
    .apt_install(
        "software-properties-common",
        "git",
        "git-lfs",
        "coreutils",
        "aria2",
        "libgl1",
        "libglib2.0-0",
        "wget",
        "libsm6",
        "libxrender1",
        "libxext6",
        "ffmpeg",
    )
    .run_commands("git clone https://github.com/lllyasviel/Fooocus.git")
    .pip_install("pygit2==1.12.2")
    .run_function(init_Fooocus, gpu=GPU_CONFIG)
)

stub = Stub("Fooocus", image=image)

@stub.function(gpu=GPU_CONFIG, timeout=60 * 10)
@web_server(port=PORT, startup_timeout=180)
def run():
    import os
    import subprocess

    os.chdir(f"/Fooocus")
    subprocess.Popen(
        [
            "python",
            "entry_with_update.py",
            "--listen",
            "127.0.0.1",
            "--port",
            "8000",
            "--always-high-vram"
        ]
    )
```

**Comment by charlesfrye** [comment]
Thanks for sharing! If you document this code and make a PR adding it under the `misc/` directory with the name `serving_fooocus.py`, we'd be happy to bring it in.

**Comment by charlesfrye** [comment]
closed in #721

**Comment by skysamuel8-debug** [comment]
hello guys how run defooocus on mondal? like this guys had do for fooocus https://github.com/BarrenWardo/Fooocus-Modal

please can you make an tutorial for defooocus i am noob
